src/Umbraco.Web.UI.Client/src/views/common/infiniteeditors/templatesections/templatesections.controller.js

```diff
@@ -1,26 +1,36 @@
 (function () {
     "use strict";
 
-    function TemplateSectionsController($scope, formHelper) {
+    function TemplateSectionsController($scope, formHelper, localizationService) {
 
         var vm = this;
 
+        vm.labels = {};
+
         vm.select = select;
         vm.submit = submit;
         vm.close = close;
 
         $scope.model.mandatoryRenderSection = false;
 
-        if(!$scope.model.title) {
+        if (!$scope.model.title) {
             $scope.model.title = "Sections";
         }
 
         function onInit() {
-            if($scope.model.hasMaster) {
+            if ($scope.model.hasMaster) {
                 $scope.model.insertType = 'addSection';
             } else {
                 $scope.model.insertType = 'renderBody';
             }
+
+            var labelKeys = [
+                "template_sectionMandatory"
+            ];
+
+            localizationService.localizeMany(labelKeys).then(function (data) {
+                vm.labels.sectionMandotory = data[0];
```

Comment:
Hi @bjarnef. I haven't tested the code yet, but spotted `sectionMandotory`. Could you change the occurences to `sectionMandatory`?

Reply:
Thanks @abjerner ... I have now fixed these typos ;)